sql/src/main/java/io/crate/protocols/postgres/PostgresWireProtocol.java

```diff
@@ -185,10 +187,14 @@
 
     private State state = PRE_STARTUP;
 
-    PostgresWireProtocol(SslReqHandler sslReqHandler, SQLOperations sqlOperations, Authentication authService) {
+    PostgresWireProtocol(SQLOperations sqlOperations, Authentication authService) {
```

Comment:
Is this constructor only used in tests? If so - would it be possible to move it?